v22.2/monitor-and-debug-changefeeds.md

```diff
@@ -7,6 +7,21 @@ docs_area: stream_data
 
 Changefeeds work as jobs in CockroachDB, which allows for [monitoring](#monitor-a-changefeed) and [debugging](#debug-a-changefeed) through the [DB Console](ui-overview.html) [**Jobs**](ui-jobs-page.html) page and [`SHOW JOBS`](show-jobs.html) SQL statements using the job ID.
 
+<a name="changefeed-retry-errors"></a>
+
+{% include_cached new-in.html version="v22.2.1" %} By default, changefeeds treat errors as retryable except for some specific terminal errors that are non-retryable.
+
+- **retryable**: The changefeed will automatically retry whatever caused the error. (You may need to intervene so that the changefeed can resume.)
+- **non-retryable**: The changefeed has encountered a terminal error and fails.
+
+The following define the categories of non-retryable errors:
+
+- When the changefeed cannot verify the target table's schema. For example, the table is offline or there are types within the table that the changefeed cannot handle.
+- The changefeed cannot convert the data to the specified [output format](changefeed-messages.html). For example, there are [Avro](changefeed-messages.html#avro) types that changefeeds do not support, or a [CDC transformation](cdc-transformations.html) is using an unsupported or malformed expression.
+- The terminal error happens as part of established changefeed behavior. For example, you have specified the [`schema_change_policy=stop` option](create-changefeed.html#schema-policy) and a schema change happens.
```

Comment:
If you like my intro sentence above, I'd then recommend introducing each bullet point identically. Example:

> - The changefeed cannot verify ...
> - The changefeed cannot convert ...
> - The changefeed has been configured with the [`schema_change_policy=stop` option], and a schema change occurs.

Seems this list is not conclusive, so there might be merit in including the concept of "other expected, established, user-requested non-retries, similar to `schema_change_policy=stop`. Not sure what these entail if any.